src/components/cc-logs/cc-logs.js

```diff
@@ -698,8 +702,11 @@ export class CcLogs extends LitElement {
       this._logsCtrl.limit = this.limit;
     }
 
-    if (changedProperties.has('filter')) {
-      this._logsCtrl.filter = this.filter;
+    if (changedProperties.has('messageFilter') || changedProperties.has('metadataFilter')) {
+      this._logsCtrl.filter = {
+        message: this.messageFilter,
+        metadata: this.metadataFilter,
+      };
```

Comment:
question: curious to know why the controller does not have two params for the filters like the component

Reply:
This setter could have two parameters. I would be ok with that.
But, for performance reasons, I want one single setter to have just one call to `this._updateList` method.